TESTING.md

```diff
@@ -247,27 +251,76 @@ Useful tips:
+   with transcripts should also disappear.
 9. Use only the keyboard. For example, navigate transcripts with arrows, delete, enter.
 10. Start typing while being focused on Chat History to perform search-by-title.
 11. Open multiple chats and ask few simultaneous questions in several sessions at once.
 12. Open new chat with <kbd>Alt</kbd> + <kbd>=</kbd> shortcut (or <kbd>Option</kbd> + <kbd>=</kbd> on Mac).
 
+## Multi-repo context
+
+### Free/pro accounts:
+
+1. Open `sourcegraph/cody` project with non-enterprise account.
+2. Open new chat and ask question about current repo (e.g. some class) - assistant should know the answer.
+3. Open new chat and ask question about squirrel - assistant should describe you an animal.
+4. Open new chat and disable local context. Ask about current repo (e.g. some class) - assistant should not have a
+   context.
+5. Save current context as default. Close the IDE. Reopen the IDE.
+    - Go to Chat History tab and open previous chats one by one. Both history and context settings are properly
+      preserved.
+    - Open new chat. Context should be disabled, as we previously set that as new default. Enable it again and set as
+      default.
+
+### Enterprise accounts:
+
+1. Open `sourcegraph/cody` project with enterprise account.
+2. Re-do all check from `Testing free/pro accounts` section but now with enterprise account.
+3. Click [+] button in the context panel and type sourcegraph repo url (`github.com/sourcegraph/sourcegraph`)
+    - Validator should block accepting incomplete or invalid URL.
+    - Add the `sourcegraph/sourcegraph` repo by hitting Add button.
+4. Open new chat and ask question about squirrel - assistant should describe you an HTTP server, **NOT** animal.
+5. Set current context as default and open new chat. It should use/display default context configuration.
+6. Disable `sourcegraph/sourcegraph` remote repo context.
+7. Ask question about squirrel. It should again describe you an animal or have no context.
+8. Save current context as default. Close the IDE. Reopen the IDE.
+    - Go to Chat History tab and open previous chats one by one. Check if both history and context settings are properly
+      preserved.
+    - Open new chat and check if `sourcegraph/sourcegraph` is disabled. It should be, as we previously set that as a
+      default. Enable it again and set as default.
+    - Open new chat and check if `sourcegraph/sourcegraph` is enabled, it should be.
+    - Remove `sourcegraph/sourcegraph` repo by selecting it, and then clicking [-] button.
+    - Ask question about squirrel. It should again describe you an animal or have no context.
+
+## Changes
+
+* Added multi-repo enterprise context
+* Improved look of the local context panel
+* Fixed several bugs like:
+    * history service being instantiated as application-wide service instead of project-wide
+    * current repo url parsing and timing issues
```

Comment:
This Changes part has been pasted by mistake i guess.

Reply:
Nope, it's valid.